app/main/views.py

```diff
@@ -87,26 +87,43 @@ def sum_team_points_by_date(team_results):
 
 def _gen_ind_standings(race_info, race_calendar):
     """Return top individual racer standings with individual race and total points
-    """
 
-    # racer_id, racer_name, race_date, points, team_id, team_name
+    Note, individual placing tiebreak is by number of wins, followed by number of
+    seconds palaces, etc.
```

Comment:
What if nobody in the ACA has a palace?

Then what happens?

Would a mansion suffice?